tools/tests/com.liferay.ide.project.core.tests/META-INF/MANIFEST.MF

```diff
@@ -24,7 +24,8 @@ Require-Bundle: org.eclipse.core.runtime,
  org.apache.commons.collections;bundle-version="3.2.0",
  org.apache.commons.lang;bundle-version="2.6.0",
  com.liferay.ide.server.core.tests;bundle-version="2.1.1",
- com.liferay.ide.core
+ com.liferay.ide.core,
+ com.liferay.ide.maven.core
```

Comment:
project.core.tests can't depend on maven.core

if we need to we may need to add tests to maven.core.tests instead.